src/models/student.model.ts

```diff
@@ -1,62 +1,95 @@
-import {ApiProperty} from '@nestjs/swagger';
-import {Column, Entity, PrimaryGeneratedColumn} from 'typeorm';
+import { ApiProperty } from '@nestjs/swagger';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Country, GenericList } from '.';
 import { Gender } from './enums';
 
+export enum EducationType {
+  PreSchool = 'PreSchool',
+  School = 'School',
+  Course = 'Course'
+}
+
```

Comment:
In the future, put this in the enum file. @RamaMahjoub

Reply:
👍🏻